client/plots/sampleView.js

```diff
@@ -50,7 +50,7 @@ class SampleView {
 		const state = this.getState(appState)
 		const q = state.termdbConfig.queries
 		const hasPlots = q ? q.singleSampleGenomeQuantification || q.singleSampleMutation : false
```

Comment:
tiny err here. if ds has none of these two then NIdata won't show. in `setControls()` simply detect optional datatypes from termdbConfig.queries and show those